<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lorem</class>
 <widget class="QWidget" name="lorem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>611</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
border : 1px solid white;
background-color: rgb(161, 161, 161);
}</string>
     </property>
     <property name="text">
      <string>Job title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>Add the totle you are hiring for</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
border : 1px solid white;
background-color: rgb(161, 161, 161);
}</string>
     </property>
     <property name="text">
      <string>Company</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
border : 1px solid white;
background-color: rgb(161, 161, 161);
}</string>
     </property>
     <property name="text">
      <string>Workplace type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>271</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_14">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>290</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
border : 1px solid white;
background-color: rgb(161, 161, 161);
}</string>
     </property>
     <property name="text">
      <string>Job location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>330</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Mashhad , Razavi khorasan , Iran</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
border : 1px solid white;
background-color: rgb(161, 161, 161);
}</string>
     </property>
     <property name="text">
      <string>Job type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>430</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>510</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_2{
border-radius:10px;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
border:1px solid black;
}
#pushButton_2:hover{
color:black;
}</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
